plugins/dimension-tag/src/test/scala/com/stratio/sparkta/plugin/dimension/tag/test/TagDimensionSpec.scala

```diff
@@ -32,25 +32,38 @@ with BeforeAndAfter
 with BeforeAndAfterAll
 with TableDrivenPropertyChecks {
 
-  val tags1 = Seq("a", "b", "c")
-  val tags2 = Seq("a", "b", "c", "a")
-  val tags3 = Seq("a", System.currentTimeMillis(), System.currentTimeMillis(), "a")
-  val tags4 = Seq("a", "a", "a", "a")
+  var tagDimension: TagDimension = null
+  before {
+    tagDimension = new TagDimension()
+  }
+
+  after {
+    tagDimension = null
+  }
 
   "A TagDimension" should {
+    "In default implementation, get 3 buckets for all precision sizes" in {
+      val buckets = tagDimension.bucket(Seq("").asInstanceOf[JSerializable]).map(_._1.id)
+
+      buckets.size should be(3)
+
+      buckets should contain(TagDimension.AllTagsName)
+      buckets should contain(TagDimension.FirstTagName)
+      buckets should contain(TagDimension.LastTagName)
+    }
+
     "In default implementation, every proposed combination should be ok" in {
-      val tagBucketer = new TagDimension()
       val data = Table(
         ("s", "rz"),
-        (tags1, tags1.size),
-        (tags2, tags2.size),
-        (tags3, tags3.size),
-        (tags4, tags4.size)
+        (Seq("a", "b", "c"), 3),
```

Comment:
The previous code was done to fix scalastyle and improve readability. Back to the previous code